### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by electricfungi**
Updated readme.md to instruct folks to use the template workflow instead of the clone workflow

### CODE REVIEW

**Comment by d-haldane** [comment]
For the git commands - set it up to use http instead of requiring ssh to be set up on the machine.

does this really work? -` git clone --recursive git@github.com:<username>/<repository>.git `

**Comment by d-haldane** [comment]
What about instructing new users to use the button on github instead of the command line /

File: README.md [hunk 1]
```diff
@@ -1,16 +1,22 @@
 # Getting started with a JITX Design
 
-Clone this template repository:
+Select the green `Use This Template` button towards the top of this page and create a new repository. If you do not see this button, sign into git.
```
Comment (bpellenard) [inline README.md:3]: I would put some highlight on new since it is the name of the button for me

Screen Shot 2021-09-09 at 3 56 49 PM

Comment (electricfungi) [reply README.md:3]: you're in the overview page for JITX. Click on design-template. This is where the _readme.md_ appears. You should see the Use this template button there.

**Review by bpellenard — APPROVED** [review]

**Pull request merged**